samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/ui/settings/SettingsViewModel.kt

```diff
@@ -56,23 +59,23 @@ class SettingsViewModel @Inject constructor(
 
     private fun validateCustomTrackUrl(trackUrl: String?): String {
```

Comment:
nit-picky, but I think using "track" in the naming here could be confusing? Since we are no longer using word "track".

Reply:
Yeah that was a miss 🤦🏻 Removed track from name here.